salt/modules/napalm_network.py

```diff
@@ -1216,6 +1228,11 @@ def load_config(filename=None,
 
         .. versionadded:: 2016.11.2
 
+    saltenv: ``base``
+        Specifies the Salt environment name.
+
+        .. versionadded:: 2017.7.3
```

Comment:
Same comment here about the version name.

Reply:
fixed